$OpenBSD: patch-CMakeLists_txt,v 1.1.1.1 2019/11/16 10:27:43 rsadowski Exp $

- Drop Git
- Stop searching for PdfLatex and disable docs
- Remove APPLICATION_SHORTNAME from install path to avoid upper-case names

Index: CMakeLists.txt
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -61,10 +61,6 @@ include(GNUInstallDirs)
 include(DefineInstallationPaths)
 include(GenerateExportHeader)
 
-include(GetGitRevisionDescription)
-
-get_git_head_revision(GIT_REFSPEC GIT_SHA1)
-
 add_definitions(
     -DQT_USE_QSTRINGBUILDER
     -DQT_MESSAGELOGCONTEXT #enable function name and line number in debug output
@@ -182,7 +178,7 @@ if(BUILD_CLIENT)
         find_package(INotify)
     endif()
     find_package(Sphinx)
-    find_package(PdfLatex)
+    #find_package(PdfLatex)
     find_package(OpenSSL 1.1 REQUIRED )
 
    find_package(ZLIB REQUIRED)
@@ -227,8 +223,8 @@ if(BUILD_CLIENT)
     add_subdirectory(src)
     if(NOT BUILD_LIBRARIES_ONLY)
         add_subdirectory(man)
-        add_subdirectory(doc)
-        add_subdirectory(doc/dev)
+	#add_subdirectory(doc)
+	#add_subdirectory(doc/dev)
         if(IS_DIRECTORY ${CMAKE_SOURCE_DIR}/admin)
             add_subdirectory(admin)
         endif(IS_DIRECTORY ${CMAKE_SOURCE_DIR}/admin)
